source/elements/oneDNN/source/conf.py

```diff
@@ -57,7 +57,8 @@
 \\newcommand{\\diffdstiterc}{\\operatorname{diff\\_dst\\_iter\\_c}}
 \\newcommand{\\diffgamma}{\\operatorname{diff\\_\\gamma}}
 \\newcommand{\\diffbeta}{\\operatorname{diff\\_\\beta}}
-\\newcommand{\\workspace}{\\operatorname{workspace}}'''
+\\newcommand{\\workspace}{\\operatorname{workspace}}
```

Comment:
I don't think this conf will be used for the full document build. I merged the PR so it will be easier to view the builds. The build is running now: https://github.com/oneapi-src/oneAPI-spec/actions/runs/3436783423. When it completes, can you check https://oneapi-src.github.io/oneAPI-spec/spec/ and https://rscohn2.github.io/oneAPI-spec/oneAPI-spec.pdf to make sure they look ok?

Reply:
Sure I will check the document there.
In the local build, those commands are properly applied and render as expected.
![image](https://user-images.githubusercontent.com/22825297/201093563-460205e1-d34f-4066-9f66-a2da898aed91.png)

Here you can see that src, dst and weights render as operatornames, whereas the other symbols render as variables (slight italic).

Reply:
This file is shared between the onednn conf and the whole spec conf

https://github.com/oneapi-src/oneAPI-spec/blob/main/source/conf/common_conf.py

Reply:
So the build finished for the html version, and I can confirm it does not render properly on the full document build.

Reply:
Thanks. I think this is not enough for proper rendering of equations in html build using mathjax (hence the above image where non of the latex commands render properly in full build).

Is it ok to update  `imgmath_latex_preamble` and `mathjax3_config` in the common_conf.py, so that those render properly for both pdf and html?